Xml.psm1

```diff
@@ -634,44 +635,48 @@ function ConvertFrom-Html {
     return $xml
 }
 
-
 function Convert-Xml {
     #.Synopsis
     #   The Convert-XML function lets you use Xslt to transform XML strings and documents.
     #.Description
     #   Documentation TODO
-    [CmdletBinding(DefaultParameterSetName="Xml")]
+    [CmdletBinding(DefaultParameterSetName="Xsl")]
```

Comment:
Just a misunderstanding.  As you discovered this commit included no breaking changes (I learned my lesson!).  Thanks for circling back.